packages/bpk-component-page-indicator/src/BpkPageIndicator.js

```diff
@@ -0,0 +1,190 @@
+    showNav,
+    totalItems,
+    updateItemCallback,
+    ...rest
+  } = props;
+  const classNames = getClassName('bpk-page-indicator', className);
+
+  const updateItem = (index) => {
+    let newItem = index;
+    if (index < 0) {
+      newItem = totalItems - 1;
+    }
+    if (index >= totalItems) {
+      newItem = 0;
+    }
+    updateItemCallback(newItem);
+  };
+
+  const handleKeyboardEvent = (e, index) => {
+    if (e.key === KEYS.ENTER || e.key === KEYS.SPACE || e.key === ' ') {
+      e.preventDefault();
+      updateItemCallback(index);
+    }
+  };
+
+  const direction = isRTL() ? 1 : -1;
+  const handlers = useSwipeable({
+    onSwipedLeft: () => updateItem(currentItem - direction),
+    onSwipedRight: () => updateItem(currentItem + direction),
+    trackMouse: true,
+    trackTouch: true,
+    preventScrollOnSwipe: true,
+  });
+
+  const bulletArray = [];
+  for (let i = 0; i < totalItems; i += 1) {
+    bulletArray.push(i);
+  }
+
+  return (
+    // $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'.
+    <div {...handlers} className={classNames} {...rest}>
+      {showNav && (
+        <BpkButton
+          iconOnly
+          link
+          onClick={() => {
+            updateItem(currentItem - 1);
+          }}
+        >
+          <AlignedLeftArrowIcon />
+          <span className="visually-hidden">Prev</span>
+        </BpkButton>
+      )}
+      <div className={getClassName('bpk-page-indicator__container')}>
+        <div
+          className={getClassName('bpk-page-indicator__bulletsContainer')}
+          style={
+            currentItem > START_SCROLL_INDEX
+              ? {
+                  '--scroll-index':
+                    totalItems > DISPLAYED_TOTAL
+                      ? Math.min(
+                          currentItem - START_SCROLL_INDEX,
+                          totalItems - DISPLAYED_TOTAL,
+                        )
+                      : 0,
+                }
+              : undefined
+          }
+        >
+          {bulletArray.map((index) => (
+            <div
+              role="button"
+              tabIndex={0}
+              onClick={() => {
+                updateItem(index);
+              }}
+              onKeyDown={(e) => {
+                handleKeyboardEvent(e, index);
+              }}
+              className={getClassName(
+                'bpk-page-indicator__bullet',
+                index === currentItem && 'bpk-page-indicator__bullet--active',
+                index === currentItem && activeItemClassName,
+                itemClassName,
+              )}
+              aria-label={`${ariaLabel} ${index + 1}`}
+            />
+          ))}
+        </div>
+      </div>
+      {showNav && (
+        <BpkButton
+          iconOnly
+          link
+          onClick={() => {
+            updateItem(currentItem + 1);
+          }}
+        >
+          <AlignedRightArrowIcon />
+          <span className="visually-hidden">Next</span>
```

Comment:
same here

Reply:
Have removed empty span and replace it with new aria-label, thank you!